django/thunderstore/cache/cache.py

```diff
@@ -158,7 +158,7 @@ def get_default(*a, **kw):
                 cache_bust_condition=self.cache_until,
                 cache_type="view",
                 key=get_view_cache_name(type(self)),
-                vary_on=args + tuple(kwargs.values()) + (self.request.community_site,),
+                vary_on=args + tuple(kwargs.values()) + (self.community_identifier,),
```

Comment:
This isn't a safe way to access it, given that `ManualCacheMixin` is not inheriting `CommunityMixin`. You'll want to do something like adding `get_extra_cache_vary` to `ManualCacheMixin` as a function that can be overwritten (defaulting to an empty set) and then create a `ManualCacheCommunityMixin` which inherits both mixins and returns the community identifier. Doing so properly separates all components from each other.